src/Php70/README.md

```diff
@@ -1,16 +1,15 @@
 Symfony Polyfill / Php70
 ========================
 
-This component provides functions unavailable in releases prior to PHP 7.0:
+This component provides features unavailable in releases prior to PHP 7.0:
 
 - [`intdiv`](http://php.net/intdiv)
 - [`preg_replace_callback_array`](http://php.net/preg_replace_callback_array)
 - [`error_clear_last`](http://php.net/error_clear_last)
 - `random_bytes` and `random_int` (from [paragonie/random_compat](https://github.com/paragonie/random_compat))
 - [`*Error` throwable classes](http://php.net/Error)
-
-Also, it provides a constant added to PHP 7.0:
 - [`PHP_INT_MIN`](http://php.net/manual/en/reserved.constants.php#constant.php-int-min)
+- `SessionUpdateTimestampHandlerInterface`
```

Comment:
everything in the list except this bullet is linkable

Reply:
I don't have any target for this, so I cannot add a link.

Reply:
damn. then it would be nice to document types of it. or it's `mixed` x3 ?

Reply:
docblocks added